.github/actions/do_build_run_sycl_e2e/action.yml

```diff
@@ -0,0 +1,80 @@
+name: build sycl e2e
+description: build sycl e2e
+
+inputs:
+  target:
+    description: 'target architecture'
+  sycl_device:
+    description: '"opencl" or "native_cpu"'
+    default: 'opencl'
```

Comment:
This probably should not have a default. You specify `sycl_device` in both places you use this action already.